src/Psl/Type/Internal/NonNullType.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Type\Internal;
+
+use Psl\Type;
+use Psl\Type\Exception\AssertException;
+use Psl\Type\Exception\CoercionException;
+
+/**
+ * @ara-extends Type\Type<nonnull>
```

Comment:
@veewee these are used by ara definition generator to generate more accurate definitions, as `nonnull` type doesn't exist in psalm :/